inthe_am/taskmanager/models/usermetadata.py

```diff
@@ -4,8 +4,8 @@
 
 
 class UserMetadata(models.Model):
-    user = models.ForeignKey(
-        User, related_name="metadata", unique=True, on_delete=models.CASCADE
+    user = models.OneToOneField(
```

Comment:
I believe this'll require that a new migration be generated -- could you generate one following django's docs for that?

Reply:
I'm not super familiar with the model, but after rollback the change and run `makemigrations`, it was created a new migration which seems not to have anything to do with this change:
```sh
Migrations for 'taskmanager':
  inthe_am/taskmanager/migrations/0009_auto_20210710_1608.py
    - Remove field store from taskstoreactivity
    - Delete model TaskStoreActivity
```
Is it ok? After that, I applied the `OneToOneField` change and the new migration was created:
```sh
Migrations for 'taskmanager':
  inthe_am/taskmanager/migrations/0010_auto_20210710_1608.py
    - Alter field user on usermetadata
```
I think it's ok to leave both.

Reply:
Ahh, yeah -- after posting that comment, I checked inthe.am to make sure there weren't any un-generated migrations, and there was one!  I've pushed up the missing migration now; so if you merge in the latest on the `development` branch you should see the new migration.  In any case, I can generate this migration for you, though, once I get around to trying to merge this.